sidebars.js

```diff
@@ -27,7 +27,48 @@ module.exports = {
     {
       type: "category",
       label: "Playbook",
-      items: [...getItems("playbook"), ...getItems("workshop")],
+      items: [
+        {
+          type: "category",
+          label: "Getting set up on the platform",
+          items: getItems("playbook/getting-set-up"),
+        },
+        {
+          type: "category",
+          label: "Administrating Users & Departments",
+          items: getItems("playbook/administrating-users-and-departments"),
+        },
```

Comment:
Did we decide that this was a sub category of "Getting set up on the platform" in the session yesterday?